x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel.tsx

```diff
@@ -372,7 +388,7 @@ export const InnerVisualizationWrapper = ({
                     })}
                   </EuiButtonEmpty>
 
-                  {localState.expandError ? visibleErrorMessage : null}
+                  {localState.expandError ? longMessage || visibleErrorMessage : null}
```

Comment:
I don't like the way this message is presented for anything but error messages. I would strongly prefer that we split this into 3 states, not 2:

1. Visualization warning with all messages visible, but not a "scary" message
2. Error state with warning triangle
  2b. Error state with expanded details